ui/library/index.ts

```diff
@@ -3,7 +3,7 @@
  */
 
 import './less/index.less';
-
+import 'react-pdf/dist/esm/Page/AnnotationLayer.css';
```

Comment:
You could also move this to `/ui/library/less/index.less`. It might be cleaner to have all the styling file imports in that file.

`@import (less) 'react-pdf/dist/esm/Page/AnnotationLayer.css';`

Reply:
Ah this is much cleaner! Will change to that.